df.refs.xml

```diff
@@ -636,27 +658,41 @@
-        <int32_t init-value='-1'/>
+        <int32_t name='land_holder' init-value='-1' comment="index into entity.positions.assignments of Civilization"/>
+        <enum name='type' type-name='entity_site_link_type' base-type='int32_t'/>
+        <int32_t name='unk_2' init-value='-1'/>
+        <int32_t name='unk_3' init-value='-1'/>
         <bitfield name='flags' type-name='entity_site_link_flags'/>
-        <int32_t comment="powers of 2 plus 5 seen. Bitfield?"/>
+        <int32_t name='unk_4' comment="Bitfield? Summary: 0 seems to be active, 1, 2, 3, 5, 131, various cessations of activity. 8, 16, 128, 144, 8192 something unrelated, and the 129, 195 possible cessations"/>
+          0: Civilization/SiteGovernment/NomadicGroup/Religion/Outcast/MerchantCompany/Guild. Seems to be 'Active'. Civ can have 'Active' claim on site not held, while others seem to be currently active.
+          1: SiteGovernment/NomadicGroup/Outcast. Seems to be defeated and losing presence at the site. Nomads/Outcasts might not have been the masters of the site as single rampage can cause enmity of two entities.
+          2: Civilization: type = None/Active, (failed to find difference). All sites seen were either destroyed or abandoned, can be reclaimed, but without that flag set. No flags set except a possible 'fortress'. Note that no conquest was seen.
+             Outcast: type = Local_Activity. All have criminal flags. Some may have indications of having left, but some don't...
+          3: Outcast, all with type = Local_Activity. All seen were fully incorporated into other criminal gangs.
+          5: SiteGovernment/MilitaryUnit. All Fortress. All type = Claim. SiteGovernment just replaced w/o event. MercenaryCompay abandoned or were replaced silently, and no flags set.
+          8: All SiteGovernment with type = None. Seems to be references to SiteGovernments without any known relations. No flags set.
+          16: SiteGovernment/NomadicGroup with type = None. Seems to be references to entities without any known relations. No flags set.
+          128: SiteGovernment/NomadicGroup/Outcast with type = None. Seems to be references to entities without any known relations. No flags set.
+          129: NomadicGroup settling in destroyed site/site horribly experimented on, being present silently, and site getting destroyed again. type = Claim.
+          131: Outcast, type = Local_Activity. No flags. All relocated to site location, typically catacombs. All fully incorporated into other organization, as per 3.
+          144: NomadicGroup (only one entry). type = None. No flags. No apparent connection.
+          195: Outcast (only 2 entries). type = Local_Activity. No flags. Both moved to inn and became primary criminal org. One had lots of members moving from the inn. Different site destroyed in both cases. No incorporation seen, though.
+          8192: Civilization/SiteGovernment. type = None. No flags. No apparent connection.
```

Comment:
A large power of 2 like this makes me think it's almost certainly a bitfield. There are others close to powers of two as well (129 = 128+1, 131 = 128+2+1). I'd just make it a bitfield and indicate somehow that you've seen up to the 14th bit (2^13) set.

Reply:
I'm fairly sure it's some kind of bitfield as well. The problem is that it's hard enough to try to make some sense out of the combinations, and decomposing those back to bits was a task I wasn't able to complete (which is why I haven't named any bits). It sort of looks like 0-5 are enums, with 8 and higher powers of 2 being bit fields with a completely different set of meanings.
3 doesn't look like a combination of 1 and 2. 131 looks like 3 with a 128 bit tacked on, but 195 doesn't look like a 3 with added weirdo bits...

As you can see, I did take the naming leap for the enum higher up.

If I were to name the bits, it would be to something like active, failure_1, failure_2, unk_4, unk_8, unk_16, unk_32, unk_64, unk_128, unk_256, unk_512, unk_1024, unk_2048, unk_4096, unk_8192. It would be butt ugly and would make little sense in itself (the comment would need to remain).

Reply:
bitfields are great for check-structures-sanity because it can tell us when new bits get added

you can just add a bunch of unnamed `<flag-bit/>`  elements to it if you don't know what they are

Reply:
As you can see further down (next commit), I did, but it ain't pretty...